apps/src/lib/kits/maker/dropletConfig.js

```diff
@@ -109,8 +110,9 @@ export const blocks = [
   {func: 'buzzer.note', category: CIRCUIT_CATEGORY, paletteParams: ['note', 'duration'], params: ['"A4"', '100'], paramButtons: { minArgs: 1, maxArgs: 2}},
   {func: 'buzzer.off', category: CIRCUIT_CATEGORY},
   {func: 'buzzer.stop', category: CIRCUIT_CATEGORY},
-  {func: 'buzzer.play', category: CIRCUIT_CATEGORY, paletteParams: ['notes', 'tempo'], params: [stringifySong(SONG_CHARGE), 120], paramButtons: { minArgs: 1, maxArgs: 2}},
```

Comment:
Note, this file defines blocks the editor recognizes, not necessarily what shows up in the toolbox. Do we still want our editor to recognize this block if it's been typed in, and/or to be able to autocomplete this in the text editor?  If so, we should leave this block definition in, but remove the block from the toolbox in another file.

Reply:
Note, we'll still need to fix up the toolboxes for individual levels.